packages/forms-web-app/src/lib/encode-string.js

```diff
@@ -1,7 +1,13 @@
 const encodeString = (stringToEncode) => {
 	if (!stringToEncode || typeof stringToEncode !== 'string') return '';
 
-	const encodedString = encodeURIComponent(stringToEncode);
+	let encodedString = '';
+
+	try {
+		encodedString = encodeURIComponent(stringToEncode);
+	} catch {
+		encodedString = stringToEncode;
```

Comment:
Would be the optimal expected behaviour to return the primary payload if it couldn't be decoded (catch error)? Doesn't it have a compounded effect when it goes through the rest of the steps?